### ISSUE TITLE
can i create application/ld+json in reactjs JSX  For seo

### ISSUE THREAD

**Issue by jaman99** [nfl/react-helmet#498]
can i create application/id+json in reatjs jsx google seo

**Comment by keplersj** [comment]
Yes! And you don't need react-helmet to do it. [react-schemaorg](https://www.npmjs.com/package/react-schemaorg) is probably what you're looking for.
